### ISSUE TITLE
Incorrect imagenet classification result with pretrained mobilenet

### ISSUE THREAD

**Issue by jjzhang10** [tensorflow/models#3159]
### System information
- **What is the top-level directory of the model you are using**:
mobilenet_v1
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **TensorFlow installed from (source or binary)**: Source
- **TensorFlow version (use command below)**: 1.4.0
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**: Titan X
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.
I want to run the mobilenet_v1 on imagenet classification. By using the pretrain weights  mobilenet_v1_1.0_224.ckpt, the result seems wrong. I used the similar code for the vgg model, the result seems correct. 

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

The code:
[test_mobilenet.py.txt](https://github.com/tensorflow/models/files/1628616/test_mobilenet.py.txt)

The classification result (top k) is : 768, 651, 762,...

The test image:
![ilsvrc2012_val_00000003](https://user-images.githubusercontent.com/11369834/34908944-4e9d6ece-f85e-11e7-847d-64d2a8a50b20.JPEG)

**Comment by jjzhang10** [comment]
Found the issue.
Add the mobilenet_v1 scope.

with slim.arg_scope(mobilenet_v1.mobilenet_v1_arg_scope()):
            logits, _ = mobilenet_v1.mobilenet_v1(images, num_classes=1001, is_training = False)

**Comment by YaoZhongtian** [comment]
Your solution worked. But why？

**Comment by jjzhang10** [comment]
slim.arg_scope add some settings to the layers, which are different with the default settings.

**Comment by YaoZhongtian** [comment]
I see. I think I can read the document to get the detail. Thanks!
